### PR TITLE
Customize CSV format

### PR DESCRIPTION
**Pull request by britaumich**
Customize CSV format for these active admin pages:
CourseAssignment
SessionAssignment
EnrollmentActivity
Enrollment

### CODE REVIEW

File: app/admin/session_assignments.rb [hunk 1]
```diff
@@ -50,9 +50,20 @@
...
+
+  csv do
+    column "Name" do |sa|
+      sa.enrollment.applicant_detail.full_name
+    end
+    column "email" do |sa|
+      sa.enrollment.user.email
+    end
+    column "Session" do |sa|
+      sa.camp_occurrence.description
```
Comment (rsmoke) [inline app/admin/session_assignments.rb:65]: What do you think about using sa.camp_occurrence.display_name instead?

File: app/admin/course_assignments.rb [hunk 2]
```diff
@@ -41,5 +41,20 @@
...
+      ca.enrollment.applicant_detail.full_name
+    end
+    column "email" do |ca|
+      ca.enrollment.user.email
+    end
+    column "Course" do |ca|
+      ca.course.title
+    end
+    column "Session" do |ca|
+      ca.course.camp_occurrence.description
```
Comment (rsmoke) [inline app/admin/course_assignments.rb:56]: What do you think about using ca.camp_occurrence.display_name instead?

**Review by rsmoke — COMMENTED** [review]
In order for the readability to be clearer I am thinking display_name may be better

File: app/admin/course_assignments.rb (hunk 2 above)
Comment (britaumich) [reply app/admin/course_assignments.rb]: You are right, of course.

File: app/admin/enrollment_activities.rb [hunk 3]
```diff
@@ -28,4 +28,19 @@
...
+      ea.enrollment.applicant_detail.full_name
+    end
+    column "email" do |sa|
+      sa.enrollment.user.email
+    end
+    column "Activity" do |sa|
+      sa.activity.description
+    end
+    column "Session" do |sa|
+      sa.activity.camp_occurrence.description
```
[the remark this answers is not in the document]
Comment (britaumich) [reply app/admin/enrollment_activities.rb]: I can't believe you did that three times.

**Review by rsmoke — APPROVED** [review]

**Pull request merged**